ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py

```diff
@@ -671,6 +671,37 @@ def test_torch_nanmean(
         keepdim=keepdims,
     )
 
+@handle_frontend_test(
+    fn_tree="torch.nansum",
+    dtype_and_x=_statistical_dtype_values(
+        function="nansum",
+        min_value=-1e04,
+        max_value=1e04,
+    ),
+    keepdims=st.booleans(),
+)
+def test_torch_nansum(
+    *,
+    dtype_and_x,
+    keepdims,
+    on_device,
+    fn_tree,
+    frontend,
+    test_flags,
+    backend_fw,
+):
+    input_dtype, x, axis = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        backend_to_test=backend_fw,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=x[0],
+        dim=axis,
+        keepdim=keepdims,
```

Comment:
Could you please add these here? Thanks!
```
...
        rtol=1e-02,
        atol=1e-02,
...
```

Reply:
Done

Reply:
Sorry, not sure if it's an issue on my side, but I don't see this change in the git diff yet 😅
The `.idea` changes looks good though, thanks!

Reply:
even i too cannot see, i think it is because of the pull request being from the previous files its not updating the recent changes, can this be merged now, thanks

Reply:
@juliagsy I fixed it can you see now

Reply:
@juliagsy any updates?